it/controllers/benefits/fuel/CompanyCarFuelBenefitsControllerISpec.scala

```diff
@@ -245,12 +241,11 @@ class CompanyCarFuelBenefitsControllerISpec extends IntegrationTest with ViewHel
     }
 
     "Update the CarFuelQuestion to no and wipe the car data when the user chooses no, redirect to company van page when prior benefits exist" which {
-
       lazy val form: Map[String, String] = Map(YesNoForm.yesNo -> YesNoForm.no)
       lazy val result: WSResponse = {
         dropEmploymentDB()
         val benefitsViewModel = aBenefitsViewModel.copy(accommodationRelocationModel = None)
-        insertCyaData(employmentUserData(isPrior = true, anEmploymentCYAModel.copy(employmentBenefits = Some(benefitsViewModel))), aUserRequest)
+        insertCyaData(anEmploymentUserData.copy(employment = anEmploymentCYAModel.copy(employmentBenefits = Some(benefitsViewModel))), aUserRequest)
```

Comment:
This copy is being done in a lot of locations. This could be a function that takes in an optional benefits view model that does all of this for you.

Reply:
That was actually the exercise, to remove the method as it is not needed.